agent/mibgroup/tlstm-mib/snmpTlstmCertToTSNTable/snmpTlstmCertToTSNTable.c

```diff
@@ -1322,16 +1322,20 @@ init_snmpTlstmCertToTSNTable_context(const char *contextName)
     }
     
     reg_oid[10] = 2;
-    to_tsn_last_changed_reg = netsnmp_create_handler_registration(
```

Comment:
It is not clear to me why the order of the netsnmp_create_handler_registration() and netsnmp_create_watcher_info() calls has been changed?

Reply:
Because `to_tsn_last_changed_reg` begins as uninitialized and is only set if the `watcher` was successfully created

Reply:
My apologies, I misread your question.
The reason why I switched the ordering of the calls to `netsnmp_create_handler_registration()` and `netsnmp_create_watcher_info()`  is because it seemed like both `to_tsn_last_changed_reg` and `watcher` are both required for the next step and that `watcher` would be the easier of the two variables to clean up if `netsnmp_create_handler_registration()` failed, because it's only a memory allocation and does not modify any external variables.